packages/integration-tests/test/interop.ts

```diff
@@ -110,7 +110,7 @@ async function createGoPeer (options: SpawnOptions): Promise<Daemon> {
   })
 
   proc.stderr?.on('data', (buf) => {
-    log.error(buf.toString())
+    log.error('interop main error - %e', buf.toString())
```

Comment:
I would defer to @achingbrain on this but I think I would have done `log.error('%e', buf.toString())` here just to keep the output similar..

also, since this is not an actual error, I don't think `%e` this is what we want in this case.